invenio_rdm_records/resources/config.py

```diff
@@ -230,6 +233,25 @@ class RDMParentRecordLinksResourceConfig(RecordResourceConfig, ConfiguratorMixin
     error_handlers = record_links_error_handlers
 
 
+#
+# Community's records
+#
+class RDMCommunityRecordsResourceConfig(RecordResourceConfig, ConfiguratorMixin):
```

Comment:
I just found out this https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/resources/resources.py#L72. Isn't this similar to the resource that is added here?